src/components/practice-button.cjsx

```diff
@@ -20,11 +21,13 @@ PracticeButton = React.createClass
     <BS.Button bsStyle="primary" className="-practice" onClick={@onClick}>{actionText}</BS.Button>
 
   onClick: ->
+    params = _.omit(@props, 'courseId')
```

Comment:
hehe, okay.  wanted to make it as flexible as possible for whatever possible properties, but i guess that does open it up a little too much